packages/runtime/plugin-runtime/src/core/server/stream/createReadableStream.worker.ts

```diff
@@ -65,6 +67,41 @@ export const createReadableStreamFromElement: CreateReadableStreamFromElement =
 
       const stream = new ReadableStream({
         start(controller) {
+          const pendingScripts: string[] = [];
+          const enqueueScript = (script: string) => {
+            if (shellChunkStatus === ShellChunkStatus.FINISH) {
+              controller.enqueue(encodeForWebStream(script));
+            } else {
+              pendingScripts.push(script);
+            }
+          };
+
+          const storageContext = storage.useContext?.();
+          const v7Active = storageContext?.activeDeferreds;
```

Comment:
Why not using `isRouterV7` in this part.

Reply:
And I think it would be best to add some comments here, such as an introduction to the differences in data between React Router v6 and v7 here. This code is quite difficult to understand.